<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectProductForm</class>
 <widget class="QWidget" name="SelectProductForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(255, 255, 255);
</string>
  </property>
  <widget class="QComboBox" name="productComboBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;  
border: 1px solid #afaaa3;  
background-color: rgb(252, 252, 252);</string>
   </property>
   <property name="placeholderText">
    <string notr="true">Ürün Seçiniz</string>
   </property>
  </widget>
  <widget class="QLabel" name="productNameLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 11px;  
background-color: rgb(255, 255, 255);
border: 1px solid #afaaa3;  </string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="startProductButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;  
border: 1px solid #afaaa3;  
background-color: rgb(252, 252, 252);</string>
   </property>
   <property name="text">
    <string>SEÇ</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;  
border: 1px solid #afaaa3;  
background-color: rgb(252, 252, 252);</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>300</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;  
border: 1px solid #afaaa3;  
background-color: rgb(252, 252, 252);</string>
   </property>
   <property name="placeholderText">
    <string>Bakiye Yükleyiniz!</string>
   </property>
  </widget>
  <widget class="QLabel" name="loadBalanceStatusLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>360</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;  
border: 1px solid #afaaa3;  
background-color: rgb(252, 252, 252);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="loadBalanceButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>330</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;  
border: 1px solid #afaaa3;  
background-color: rgb(252, 252, 252);</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>459</x>
     <y>10</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icons/icons/logo2.jpeg);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
